internal/servers/controller/handlers/marshler.go

```diff
@@ -0,0 +1,19 @@
+package handlers
+
+import (
+	"github.com/grpc-ecosystem/grpc-gateway/v2/runtime"
+	"google.golang.org/protobuf/encoding/protojson"
+)
+
+var JSONMarshaler = &runtime.JSONPb{
```

Comment:
why not just provide a function that returns the needed runtime.JSONPb?  Is there some reason all of them need to share the same instance?

Reply:
Thank you!